### ISSUE TITLE
Support re-tagging docker image

### ISSUE THREAD

**Issue by Yajo** [ansible-collections/community.docker#147]
##### SUMMARY

I need to retag an image, or to issue this command, basically: `docker image tag $source $destination`. No way to do it with `community.docker.docker_image` AFAICS.

The purpose is to roll back to an older image after a failed deployment.

##### ISSUE TYPE
- The `source` argument doesn't allow specifying a different image as source.
- A new `local_source` may help.

##### COMPONENT NAME

docker_image

##### ADDITIONAL INFORMATION

```yaml
- name: re-tag old image
  community.docker.docker_image:
    source: local
    local_source: "{{ old_image }}"
    force_tag: yes
    name: deployed/image/name:tag
```

**Comment by felixfontein** [comment]
See the examples from docker_image:
```.py
- community.docker.docker_image:
    source: local
    name: '{{ old_image }}'
    repository: deployed/image/name:tag
    force_tag: true
```

**Comment by Yajo** [comment]
Ah sorry, in
https://docs.ansible.com/ansible/latest/collections/community/docker/docker_image_module.html
says "Use with state present to tag the image into the repository."

I didn't want to tag it into the repository, but in the local docker server.

Maybe I misunderstood the docs?

**Comment by felixfontein** [comment]
The docs are not very clear, and naming that parameter `repository` is very misleading. If you don't tell the module to push, `repository` allows you to simply tag the image.

**Comment by Yajo** [comment]
Oh but it seems like `name` does not allow an image ID, right? CC @joao-p-marques

![image](https://user-images.githubusercontent.com/973709/120819278-4f626680-c54b-11eb-884e-f84f12404e69.png)

**Comment by felixfontein** [comment]
Hmm, I think the documentation is wrong here. `source=local` should also work with image IDs.

**Comment by felixfontein** [comment]
Actually, it did not work. But I found a simple fix to make it work, see #149. That also improves the docs (I hope).
